README.md

```diff
@@ -1,33 +1,108 @@
+This step can take at least one hour or more.
+
+System is ready for use, by default DHCP is tried on the first network device.
+
+Login via ssh. Use the user **dragon** with the password **password**.
+
+```bash
+ssh dragon@IpFromYourServer
+```
+
+Copy the `/home/dragon/wireguard-client.conf` file to your device. This is necessary for using the web endpoints on your device.
+Rename the wireguard config file to something like `ciab.conf`.
+> **Note:**
+> Be aware that the name does not contain any special characters!
+
+```bash
+scp dragon@IpFromYourServer:/home/dragon/wireguard-client.conf /home/ubuntu/ciab.conf
+```
+
+Install wireguard on your device, if you have not done this before. For instructions how to do it on your device, please have
+a look on the documentation of your used distribution.
+
+Start the wireguard tunnel.
+
+```bash
+wg-quick up /home/ubunu/ciab.conf
+```
+
+Now your Cloud in a Box is up and usable via the following URLs:
+
+| Name              | URL                                           | Username  | Password                                 |
+|-------------------|-----------------------------------------------|-----------|------------------------------------------|
+| ARA               | <https://ara.services.in-a-box.cloud>         | ara       | password                                 |
+| Flower            | <https://flower.services.in-a-box.cloud>      | -         | -                                        |
+| Grafana           | <https://api.in-a-box.cloud:3000>             | admin     | password                                 |
+| Homer             | <https://homer.services.in-a-box.cloud>       | -         | -                                        |
+| Horizon Admin     | <https://api.in-a-box.cloud>                  | admin     | password                                 |
+| Horizon Test-User | <https://api.in-a-box.cloud>                  | test      | test # FIX ME                            |
+| Kibana            | <https://api.in-a-box.cloud:5601>             | kibana    | password                                 |
+| Netbox            | <https://netbox.services.in-a-box.cloud>      | admin     | password                                 |
+| Netdata           | <http://manager.systems.in-a-box.cloud:19999> | -         | -                                        |
+| phpMyAdmin        | <https://phpmyadmin.services.in-a-box.cloud>  | root      | password                                 |
+| RabbitMQ          | <https://api.in-a-box.cloud:15672>            | openstack | BO6yGAAq9eqA7IKqeBdtAEO7aJuNu4zfbhtnRo8Y |
+
+> **Note:**
+> Watch out for http and https. It is necessary to use the right one.
+
+## Configure Openstack-Client
+
+Connect to your Cloud in a Box via ssh (see command above).
+Select one of the preconfigured environments `system`, `admin`, `test` by exporting the environment variable:
+
+```bash
+export OS_CLOUD=admin
+openstack server list
+```
+
```

Comment:
Should we mention how to grab the clouds.yaml/secure.yaml files from the manager? Or how to get it from the horizon dashboard?
Because eventually, you may want to access the OpenStack APIs without logging in to the manager ...

Reply:
This is out of scope for the getting started guide. This might be something for an advanced usage guide. But this is than a duplicate of the testbed / full installation.